src/Microsoft.Azure.SignalR.Management/HubInstanceFactories/IServiceHubLifetimeManagerFactory.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.SignalR.Management
+{
+    internal interface IServiceHubLifetimeManagerFactory
+    {
+        Task<IServiceHubLifetimeManager> Create(string hubName, CancellationToken cancellationToken);
+
+        Task<IServiceHubLifetimeManager> Create(string hubName, CancellationToken cancellationToken, ILoggerFactory loggerFactory);
```

Comment:
since the return value is task, change method name to CreateAsync

Reply:
This file is removed now.